client/webserver/site/src/js/register.js

```diff
@@ -107,76 +107,53 @@ export default class RegistrationPage extends BasePage {
       return
     }
     app.setLogged(true)
```

Comment:
`setLogged` checks `app.user.exchanges`, so you need to force update `app.user` with `await app.fetchUser()` before calling `setLogged` here or else the `marketsMenuEntry` won't show.

![image](https://user-images.githubusercontent.com/6109680/81619380-12ab5680-93af-11ea-8a77-16d355010d61.png)

Reply:
This is the `setAppPass()` function; (ideally, typically) only called when the app is not already initialized. Not sure there'd be anything new to discover by calling `app.fetchUser()` at this point. Also, the user would not have any DEX connections yet, so the `marketsMenuEntry` should remain hidden.

Reply:
Ah. My bad. I see what's going on. You need a `setLogged` after the `fetchUser` in `registerDEX` at [this line](https://github.com/itswisdomagain/dcrdex/blob/24a0b3a5fe24b09f296253428ef63802d8f1d1dc/client/webserver/site/src/js/register.js#L166).